src/vivarium_csu_hypertension_sdc/components/disease.py

```diff
@@ -1,7 +1,119 @@
+import numpy as np
+import pandas as pd
+from vivarium.framework.state_machine import Trigger, Transition
+from vivarium_public_health.disease import DiseaseState, DiseaseModel, SusceptibleState, RateTransition
 
 
+class RelapseTransition(RateTransition):
+    def load_transition_rate_data(self, builder):
+        if 'relapse_rate' in self._get_data_functions:
+            rate_data = self._get_data_functions['relapse_rate'](self.output_state.cause, builder)
+            pipeline_name = f'{self.output_state.state_id}.relapse_rate'
```

Comment:
risk effect will try to filter RR data to relapse rate :( 
https://github.com/ihmeuw/vivarium_public_health/blob/develop/src/vivarium_public_health/risks/data_transformations.py#L176